t/02-get.t

```diff
@@ -2470,3 +2470,71 @@ in negative callback
 in negative callback
 --- no_error_log
 [error]
+
+=== TEST 51: get() cached only
```

Comment:
Please make sure to respect the formatting here (3 line jumps between tests block and proper test number). OpenResty has a `reindex` script in devel-utils for this.